pkg/testutil/testutil.go

```diff
@@ -340,3 +350,88 @@ func BlueprintWithConfigMap(bp *crv1alpha1.Blueprint) *crv1alpha1.Blueprint {
 	}
 	return bp
 }
+
+func CreateSecret(cli kubernetes.Interface, namespace, name string, secrettype v1.SecretType, data map[string][]byte) (se *v1.Secret, err error) {
+	secret := &v1.Secret{
+		ObjectMeta: metav1.ObjectMeta{
+			GenerateName: name,
+		},
+		Data: data,
+	}
+	if secrettype != "" {
+		secret.Type = secrettype
+	}
+
+	se, err = cli.CoreV1().Secrets(namespace).Create(context.Background(), secret, metav1.CreateOptions{})
+	return
```

Comment:
do we not want to handle err here?

Reply:
That is the last line of the function, if we get an error, error would be returned from the function else `nil`
It doesnt make sense to add below lines in this case
```
if err!=nil{
return nil, err
}
```